### DISCUSSION TITLE
Multi-line text support

### DISCUSSION THREAD

**Discussion by glutanimate** [SVG-Edit/svgedit#789] [New feature request]
Sorry if this is a duplicate, but I wasn't able to find any existing issue report in this repo.

Looking at [an old feature request from Google Code](https://github.com/kahunamoore/svg-edit/issues/242), it appears like multi-line text labels where in the works sometime around 2011. The result of that seems to have been the [textedit branch](https://github.com/SVG-Edit/svgedit/tree/textedit) which is still available in this repo. Unfortunately it seems like it has been abandoned for almost 7 years now.

I was wondering what the current status on this feature was. Is multi-line text support still on the roadmap for SVG-Edit?

Thanks a lot in advance.

**Comment by robertcck** [comment]
https://github.com/SVG-Edit/svgedit/tree/e0539e66b87633a983a40e3bf0f7cb8c5a5fe4f8

Look at this commit for multi line

**Comment by sachin-rebelute** [comment]
We are also facing the same issue. Anyone know how to solve. We installed the same repository but text element is not working as multiline.

Thanks in advance

**Comment by jlpoolen** [comment]
As a work-around, see **tspan** elements that are within **text** elements https://www.w3.org/TR/SVG/text.html#TSpanElement
`<tspan dy="1em">my first line</tspan>`
`<tspan dy="1em">my second line</tspan>`

**Comment by ile** [comment]
Hello,

This would be a very useful feature. 

With `tspan` elements it wouldn't be very complicated to do either.

**Comment by tobei** [comment]
I have the same problem as I use the editor to edit "slides" in svg and let people do some light editing before submitting their work back to server. However any svg created from powerpoint/ai/designer contianing multiple line text can't be edited at the moment, nor be created.
